bin/test-workflow/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM google/cloud-sdk
+
+RUN mkdir -p /src
+WORKDIR /src
+
+# Install Docker client
+RUN apt-get update -y && \
+    apt-get install -y apt-transport-https ca-certificates curl gnupg2 software-properties-common wget && \
+    curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg | apt-key add - && \
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") $(lsb_release -cs) stable" && \
+    apt-get update && \
+    apt-get install -y docker-ce && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install kubectl CLI
+ARG KUBECTL_CLI_URL
+RUN wget -O /usr/local/bin/kubectl ${KUBECTL_CLI_URL:-https://storage.googleapis.com/kubernetes-release/release/v1.7.6/bin/linux/amd64/kubectl} && \
```

Comment:
We should at least update the default kubectl version to `1.20.9` or `1.21.3`.
And maybe use the command from the latest kubectl install documentation (even if the new command gets redirected to `storage.googleapis.com`. That would look like this:
```
curl -LO "${KUBECTL_CLI_URL:-https://dl.k8s.io/release/v1.21.0/bin/linux/amd64/kubectl}" && \
chmod +x kubectl && \
mv kubectl /usr/local/bin/kubectl
```
 Or, maybe even better, if we simplify things and just look for a Kubernetes version:
 ```
 curl -LO https://dl.k8s.io/release/v"${KUBECTL_VERSION:-1.21.0}"/bin/linux/amd64/kubectl && \
chmod +x kubectl && \
mv kubectl /usr/local/bin/kubectl
```

Reply:
Updated to the second version, and updated `secrets.yml` to account for this.